minifi/minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/service/MiNiFiCommandSender.java

```diff
@@ -61,6 +61,7 @@ public Optional<String> sendCommand(String cmd, Integer port, String... extraPar
             LOGGER.debug("Connecting to MiNiFi instance");
             socket.setSoTimeout(SOCKET_TIMEOUT);
             socket.connect(new InetSocketAddress("localhost", port), CONNECTION_TIMEOUT);
+            socket.setSoTimeout(SOCKET_TIMEOUT);
```

Comment:
Do we need both calls to `setSoTimeout`?

Reply:
To be honest I'm not sure. In the old MiNiFi implementation there were both setting. In the current RunNiFi.sendRequest code the same pattern is followed similarly to NiFiRegistry. This got removed at a refactor when we moved code here. However in another place I know @ferencerdei removed the second setSoTimeout and it caused issues so it was added back. Here I haven't faced any particular issues explicitly because of this just added it back for the sake of consistency with the other places in the code. If you feel strongly about it I can remove of course.